Library/Homebrew/mktemp.rb

```diff
@@ -40,7 +46,14 @@ def to_s
   end
 
   def run
-    @tmpdir = Pathname.new(Dir.mktmpdir("#{@prefix.tr "@", "AT"}-", HOMEBREW_TEMP))
+    if @retain_in_cache
+      source_dir = "#{HOMEBREW_CACHE}/Sources/#{@prefix.tr "@", "AT"}"
```

Comment:
Extract `#{@prefix.tr "@", "AT"}` into a variable so it can be reused below.

Reply:
Done